_translator-files/po/fr/Installation-for-iOS.po

```diff
@@ -155,4 +155,4 @@ msgstr "Jamulus a été installé et peut être utilisé maintenant. Vous pouvez
 #. type: Plain text
 #: ../wiki/en/Installation-for-iOS.md:50
 msgid "[Jamulus setup page](Getting-Started){: .button}"
-msgstr "[Page d'accueil de Jamulus](Getting-Started){: .button}"
+msgstr "[Page de démarrage de Jamulus](Getting-Started){ : .button}"
```

Comment:
Is adding the space between `{` and `:` correct?

Reply:
No, it's not. Good catch. It was probably auto-added by Weblate (because it's a French punctuation rule - but id doesn't apply here) and I didn't corrected it. I removed it.